### PR TITLE
[core] Temporarily fallback to v5 of Material UI

### PR DESCRIPTION
**Pull request by brijeshb42**
1. Fixes the issues with the current build failing for the latest beta of @mui/material package. To be investigated separately (to unblock release) since it does not directly concern Pigment CSS.
2. Fixed some TS errors coming from emotion.
3. Updated WyW to latest version.

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/pigment-css/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: apps/pigment-css-next-app/next.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
 // eslint-ignore-next-line import/no-unresolved
 const { withPigment } = require('@pigment-css/nextjs-plugin');
-const { extendTheme } = require('@mui/material/styles');
+const { experimental_extendTheme: extendTheme } = require('@mui/material/styles');
```
Comment (siriwatknp) [inline apps/pigment-css-next-app/next.config.js:4]: Is this part of the root cause that the CI fails?

Comment (siriwatknp) [reply apps/pigment-css-next-app/next.config.js:4]: Ah, you changed to v5.

**Review by siriwatknp — APPROVED** [review]

**Review by siriwatknp — CHANGES_REQUESTED** [review]
Hey, wait. v5 does not have `stringifyTheme`.

**Comment by brijeshb42** [comment]
> Hey, wait. v5 does not have `stringifyTheme`.

That does not concern the app in this repo. I have to investigate more to pinpoint the exact version of Material UI that is causing the issue. But I don't want to block release for that.

@siriwatknp

File: apps/pigment-css-next-app/package.json [hunk 2]
```diff
@@ -10,14 +10,8 @@
...
-    "@mui/utils": "next",
-    "@mui/base": "next",
-    "@mui/lab": "next",
-    "@mui/material": "next",
-    "@mui/system": "next",
-    "@mui/material-nextjs": "next",
-    "@mui/icons-material": "next",
-    "@emotion/cache": "latest",
+    "@mui/material": "^5.16.6",
+    "@mui/material-nextjs": "^5.16.6",
```
Comment (siriwatknp) [inline apps/pigment-css-next-app/package.json:14]: Can you change to `6.0.0-beta.2` if the latest beta fails.

Comment (siriwatknp) [reply apps/pigment-css-next-app/package.json:14]: It does not make sense to use v5.

**Review by siriwatknp — APPROVED** [review]

File: apps/pigment-css-next-app/package.json (hunk 2 above)
Comment (brijeshb42) [reply apps/pigment-css-next-app/package.json:14]: beta.2 also has the same issue.

**Pull request merged**